spec.html

```diff
@@ -20557,6 +21132,9 @@ <h1>Static Semantics: Early Errors</h1>
         <li>
           It is a Syntax Error if PrototypePropertyNameList of |ClassElementList| contains more than one occurrence of *"constructor"*.
         </li>
+        <li>
+          It is a Syntax Error if PrivateBoundIdentifiers of |ClassElementList| contains any duplicate entries, unless the name is used once for a getter and once for a setter and in no other entries.
```

Comment:
This is not sufficiently precise. Even if you know what "getter" and "setter" are, this would allow `class { get #x(){} static set #x(){} }`, which it should not.